using-mobile-extensions/adobe-target/README.md

```diff
@@ -427,28 +569,57 @@ The SDK can only collect information from the launching Activity if [`setApplica
 {% endhint %}
 {% endtab %}
 
-{% tab title="iOS" %}
-#### Syntax
-
-```text
-+ (void) collectLaunchInfo: (nonnull NSDictionary*) userInfo;
+{% tab title="iOS (AEP 3.x)" %}
+### Swift
+**Syntax**
+```swift
+public static func collectLaunchInfo(_ userInfo: [String: Any])
 ```
 
-#### Examples
-
-Here are some examples in Objective-C and Swift:
-
-**Objective-C**
+**Examples**
+```swift
+   let launchInfo = [
+       "key_str":"stringValue",
+       "adb_deeplink":"abc://myawesomeapp?some=param&some=other_param",
+       "adb_m_id":"awesomePushMessage",
+       "adb_m_l_id":"happyBirthdayNotification"]
+    MobileCore.collectLaunchInfo(launchInfo)
+```
 
-```text
-[ACPCore collectLaunchInfo: @{@"adb_deeplink":@"com.adobe.targetpreview://app.adobetarget.com?at_preview_token=tokenFromTarget"}];`
+### Objective C
```

Comment:
Missed one here!

Reply:
Thanks for pointing out. I have pushed the fix.